### DISCUSSION TITLE
What privacy data is collected by self-hosted version?

### DISCUSSION THREAD

**Discussion by marekbrze** [Budibase/budibase#10699] [General]
I want to use Buddibase for processing my subscribers/customers data. I know that sometimes self-hosted software still uses tools like sentry to catch bugs etc and they store data like ip addresses and forms data. 

Are there any analytics tools used by the self-hosted Buddibase software?

**Comment by shogunpurple** [comment]
Hey @marekbrze,

We do use sentry in the self hosted version of budibase for bugs. In terms of other events, you can find more information about our analytics and our event catalogue here:

https://docs.budibase.com/docs/analytics

Analytics are of course opt out, and can be controlled in the budibase portal.

**Reply by marekbrze** [reply]
Thank you so much for the answer! I can see that docs are super detailed 🙌
